README.md

```diff
@@ -165,6 +165,21 @@ import { init } from 'onfido-sdk-ui/dist/onfidoAuth.min.js'
 var Onfido = require('onfido-sdk-ui/dist/onfidoAuth.min.js')
 ```
 
+In addition to the alternative way of importing Auth, you should also set the following code in the loader (Webpack) of your application:
+
+```javascript
+new CopyPlugin({
+  patterns: [
+    {
+      from: `../../node_modules/onfido-sdk-ui/dist/auth-sdk`,
+      to: `${__dirname}/bin/src/auth-sdk`,
+    },
+  ],
+})
```

Comment:
This would only work for Webpack. For any other implementation I think it's worth mentioning that they need to create a folder called `auth-sdk` in the public assets folder of their application and copy the files from `node_modules/onfido-sdk-ui/dist/auth-sdk` into it.